### ISSUE TITLE
Add API for Safari Share window

### ISSUE THREAD

**Issue by idna001** [idna001/Couple-Memory#95]
If youre achive a highscore there is the ability to share the achievement. 
By pressing sharing on a mobile device the share Button from the browser should come up. 

This is partly done, but not functional

Thanks for contributing

**Comment by idna001** [comment]
https://github.com/idna001/Couple-Memory/issues/44

**Comment by tchesa** [comment]
This is already there. Maybe we can close this.

**Comment by idna001** [comment]
@tchesa yep, sadly is working on every testcase. Safari shared window don't work yet
